apps/app/src/pages/account/saved/index.tsx

```diff
@@ -0,0 +1,159 @@
+import { Box, Card, Center, Divider, Grid, Group, Loader, Overlay, Stack, Text, Title } from '@mantine/core'
+import { type GetServerSideProps } from 'next'
+import dynamic from 'next/dynamic'
+import { useRouter } from 'next/router'
+import { useSession } from 'next-auth/react'
+import { useTranslation } from 'next-i18next'
+import { type MouseEvent, useCallback } from 'react'
+
+import { getServerSession } from '@weareinreach/auth'
+import { Link } from '@weareinreach/ui/components/core/Link'
+import { useCustomVariant } from '@weareinreach/ui/hooks/useCustomVariant'
+import { api } from '~app/utils/api'
+import { getServerSideTranslations } from '~app/utils/i18n'
+import { SavedResultLoading } from '~ui/components/core/Saved/SavedOrgResultCard'
+import { Icon } from '~ui/icon'
+import { CreateNewList } from '~ui/modals'
+
+const QuickPromotionModal = dynamic(() =>
+	import('@weareinreach/ui/modals/QuickPromotion').then((mod) => mod.QuickPromotionModal)
+)
+
+const formatDate = (dateString: Date) => {
+	const date = new Date(dateString)
+	const daySuffix = (d: number) => {
+		if (d > 3 && d < 21) return 'th'
+		switch (d % 10) {
+			case 1:
+				return 'st'
+			case 2:
+				return 'nd'
+			case 3:
+				return 'rd'
+			default:
+				return 'th'
+		}
+	}
+	const day = date.getDate()
+	const dayWithSuffix = day + daySuffix(day)
+	const formattedDate = `${date.toLocaleDateString('en-US', { weekday: 'short' })}, ${date.toLocaleDateString('en-US', { month: 'long' })} ${dayWithSuffix}, ${date.getFullYear()}`
+	return formattedDate
+}
+
+const SavedLists = () => {
+	const { t } = useTranslation('common')
+	const { data: session, status } = useSession()
+	const router = useRouter()
+	const { data: allSavedLists, isLoading, refetch } = api.savedList.getAll.useQuery()
+	const handleReturnHome = useCallback(() => router.replace('/'), [router])
+
+	const deleteMutation = api.savedList.delete.useMutation({
+		onSuccess: () => {
+			refetch()
+		},
+	})
+
+	const handleDelete = (e: MouseEvent<HTMLButtonElement, globalThis.MouseEvent>, id: string) => {
+		e.stopPropagation()
+		deleteMutation.mutate({ id })
+	}
+
+	if (status === 'loading') {
+		return (
+			<Center>
+				<Loader />
+			</Center>
+		)
+	}
+	if (status === 'unauthenticated' || session === null) {
+		return (
+			<Overlay blur={2}>
+				<QuickPromotionModal component='button' autoLaunch onClose={handleReturnHome} />
+			</Overlay>
+		)
+	}
+
+	return (
+		<Grid.Col xs={12} sm={12}>
+			<Stack>
+				<Title order={1}> {t('list.saved')} </Title>
+				<Text size='lg'>{t('list.create-new-sub')}</Text>
+				<CreateNewList
+					style={{
+						fontSize: '16px',
```

Comment:
move styles in to a useStyles() hook - see other components for how `createStyles()` is used from `@mantine/core`